packages/models/src/types.ts

```diff
@@ -1,15 +1,10 @@
 import { LiteralUnion } from './types.utils';
 
-export type Blockchains = LiteralUnion<'bitcoin' | 'stacks', string>;
+export type Blockchain = LiteralUnion<'bitcoin' | 'stacks', string>;
 
-export type CryptoAssetType =
-  | 'btc'
-  | 'stx'
-  | 'brc-20'
-  | 'inscription'
-  | 'rune'
-  | 'sip-9'
-  | 'sip-10'
-  | 'src-20'
-  | 'stamp'
-  | 'stx-20';
+export type AccountIdentifier = 'ns' | 'tr' | 'bns' | 'stacks';
```

Comment:
We already have `AccountId` referring to an object that specifies identifies an account.

As this is a purely presentational concern (that we'll deal with much less frequently), maybe a longer more descriptive name would help avoid any confusion. Something like `AccountDisplayIdentifier` or `UserAccountIdPreference`. Ugly name here is okay.

Reply:
*in my un merged PR

Reply:
Ok, yep, was concerned about that bc at first I actually had `accountId` which def seemed problematic. I'll change.

Reply:
`AccountDisplayPreference` might work?

Reply:
Changed it to `AccountDisplayPreference` except for I left the component name as `AccountIdentifierSheet` and `AccountIdentifierCell` so it is still coordinated with designs.